src/tests/baseservices/exceptions/stackoverflow/stackoverflowtester.cs

```diff
@@ -71,12 +69,13 @@ static bool TestStackOverflow(string testName, string testArgs, out List<string>
             return true;
         }
 
-        static bool TestStackOverflowSmallFrameMainThread()
+        [Fact]
+        public static bool TestStackOverflowSmallFrameMainThread()
```

Comment:
`[Fact]` returning `bool` probably should be rejected by the wrapper generation.  (I'm guessing that the value will be ignored today.)

Reply:
Thanks for pointing that out, I have updated the ```[Fact]``` methods in the ```stackoverflowtester.cs``` source file, according to your previous comments I think you might like the additional delta.